src/_pytest/deprecated.py

```diff
@@ -32,6 +32,9 @@
     "Please update to the new name.",
 )
 
+YIELD_FIXTURE = PytestDeprecationWarning(
+    "@pytest.yield_fixture is deprecated.\nUse @pytest.fixture instead."
```

Comment:
The bit about being safe to search/replace might be useful here. 😁

Reply:
Trying to keep it short, I added "they are the same".